### PR TITLE
#788 Universal settings

### PR DESCRIPTION
**Pull request by hoangdat**
### Mobile
https://user-images.githubusercontent.com/6462404/186378869-545d09dc-9ae0-477e-bf0a-bf2e080f083e.mp4

### Web has no impact:
https://user-images.githubusercontent.com/6462404/186380366-e5736f5e-f624-4d99-b3e3-d61a3b3a58bf.mov

### Tablet:
https://user-images.githubusercontent.com/6462404/186380795-7e4a9618-497c-49c3-80a9-9e88f8052c07.mov

### CODE REVIEW

**Comment by dab246** [comment]
What about the web?

File: lib/features/manage_account/presentation/menu/widgets/setting_first_level_tile_builder.dart [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      onTap: clickAction,
+    );
+  }
+
+  Widget _buildSettingIcon(BuildContext context) {
+    return SvgPicture.asset(settingIcon, height: 24, width: 24, fit: BoxFit.fill);
+  }
+
+  double _getSubtitleLeftPadding(BuildContext context) {
+    return SettingsUtils.getHorizontalPadding(context, _responsiveUtils) + 12 + 24;
```
Comment (dab246) [inline lib/features/manage_account/presentation/menu/widgets/setting_first_level_tile_builder.dart:91]: I don't like how the operator is written like that. It doesn't make sense

**Comment by dab246** [comment]
On tablets, the padding on both sides is not equal

Screen Shot 2022-08-24 at 16 28 07

**Comment by hoangdat** [comment]
update for tablet

https://user-images.githubusercontent.com/6462404/186460597-62600520-26e9-44df-b2de-9b201c4724ac.mp4

**Comment by hoangdat** [comment]
Update for tablet

https://user-images.githubusercontent.com/6462404/186462227-399ee908-e5d5-4186-a2b2-c9674cfd894f.mov

**Review by dab246 — APPROVED** [review]
/lgtm

**Pull request merged**